### DISCUSSION TITLE
Does memcached-store require "CPU is always allocated" in Google Cloud Run?

### DISCUSSION THREAD

**Discussion by ngdaddikar** [express-rate-limit/rate-limit-memcached#10] [Q&A]
We use Google Cloud Run and I was wondering if any Memcached operations are performed in the background. For example - if there is a thread (using setTimeout) that is used to expire keys. If yes, then I will have to turn on the "CPU is always allocated" setting for our containers. If I don't do that the background function may never be called. 

Has anyone had any experience with this?

Thanks.

**Comment by nfriedly** [comment]
I don't think you need to turn on "CPU is always allocated". The store relies on memcached's built-in expiration setting for old keys.

I've never tried it with Google Cloud Run - you should try it out and see if it works, and then let us know.
